.travis.yml

```diff
@@ -5,82 +5,94 @@ language: haskell
 git:
   depth: 5
 
-
 cache:
   directories:
+  - "$HOME/.cabal"
+  - "$HOME/.ghc"
   - "$HOME/.stack"
-  - "$HOME/build/serokell/universum/.stack-work"
-
+  - "$HOME/build/kowainik/universum/.stack-work"
```

Comment:
You could use `$TRAVIS_BUILD_DIR` here. It also will help in case of renaming because we won't need to change name at least here.